.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,265 @@
+    Analyzes and prints the files that contain suppression for any custom
+    lint rule
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    """
+    print(
+        subprocess.check_output(["git", "branch"]).decode("utf-8").strip("\n")
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)